jean_zay/submitit/understanding-unets/exact_recon_unet_training.py

```diff
@@ -0,0 +1,47 @@
+from learning_wavelets.training_scripts.exact_recon_unet_training import train_unet
+from learning_wavelets.evaluation_scripts.exact_recon_unet_evaluate import evaluate_unet
+
+import numpy as np
+from scipy.stats import ttest_ind
+
+from jean_zay.submitit.general_submissions import train_eval_grid
+
+
+job_name = 'exact_recon_unet_training'
+n_epochs = 100
+batch_size = 8
+base_n_filters = 4
+n_layers = 4 
+non_linearity = 'relu'
+n_gpus = 1
```

Comment:
```suggestion
n_gpus = 2
```

Let's make it 2 for training from now on, it should be faster